tasks/eva_2406/tests/test_gather_release_species.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
I think this and the next file must've slipped in from your other ticket!

Reply:
Indeed 😊